packages/asset-buyer/docs/reference.mdx

```diff
@@ -0,0 +1,1352 @@
+`orders` | `SignedOrder`[] | - | A non-empty array of objects that conform to SignedOrder. All orders must have the same makerAssetData and takerAssetData (WETH). |
+`options` | `Partial<AssetBuyerOpts>` |  {} | Initialization options for the AssetBuyer. See type definition for details.  |
+
+**Returns:** *[AssetBuyer](#class-assetbuyer)*
+
+An instance of AssetBuyer
+
+___
+
+### `Static` getAssetBuyerForStandardRelayerAPIUrl
+
+▸ **getAssetBuyerForStandardRelayerAPIUrl**(`supportedProvider`: `SupportedProvider`, `sraApiUrl`: string, `options`: `Partial<AssetBuyerOpts>`): *[AssetBuyer](#class-assetbuyer)*
+
+*Defined in [asset_buyer.ts:73](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/asset_buyer.ts#L73)*
+
+Instantiates a new AssetBuyer instance given a [Standard Relayer API](https://github.com/0xProject/standard-relayer-api) endpoint
+
+**Parameters:**
+
+Name | Type | Default | Description |
+------ | ------ | ------ | ------ |
+`supportedProvider` | `SupportedProvider` | - | The Provider instance you would like to use for interacting with the Ethereum network. |
+`sraApiUrl` | string | - | The standard relayer API base HTTP url you would like to source orders from. |
+`options` | `Partial<AssetBuyerOpts>` |  {} | Initialization options for the AssetBuyer. See type definition for details.  |
+
+**Returns:** *[AssetBuyer](#class-assetbuyer)*
+
+An instance of AssetBuyer
+
+<hr />
+
+> # Class: InsufficientAssetLiquidityError
+
+Error class representing insufficient asset liquidity
+
+## Hierarchy
+
+* `Error`
+
+  * **InsufficientAssetLiquidityError**
+
+## Index
+
+### Constructors
+
+* [constructor](#constructor)
+
+### Properties
+
+* [amountAvailableToFill](#amountavailabletofill)
+* [message](#message)
+* [name](#name)
+* [stack](#optional-stack)
+* [Error](#static-error)
+
+## Constructors
+
+###  constructor
+
+\+ **new InsufficientAssetLiquidityError**(`amountAvailableToFill`: `BigNumber`): *[InsufficientAssetLiquidityError](#class-insufficientassetliquidityerror)*
+
+*Defined in [errors.ts:12](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/errors.ts#L12)*
+
+**Parameters:**
+
+Name | Type | Description |
+------ | ------ | ------ |
+`amountAvailableToFill` | `BigNumber` | The amount availabe to fill (in base units) factoring in slippage  |
+
+**Returns:** *[InsufficientAssetLiquidityError](#class-insufficientassetliquidityerror)*
+
+## Properties
+
+###  amountAvailableToFill
+
+• **amountAvailableToFill**: *`BigNumber`*
+
+*Defined in [errors.ts:12](https://github.com/0xProject/0x-monorepo/blob/6dd77d5c8/packages/asset-buyer/src/errors.ts#L12)*
+
+The amount availabe to fill (in base units) factoring in slippage.
+
+___
+
+###  message
+
+• **message**: *string*
+
+*Inherited from void*
+
+Defined in /Users/rickmorty/Documents/projects/0x/0x-monorepo/node_modules/typedoc/node_modules/typescript/lib/lib.es5.d.ts:974
```

Comment:
We should skip the `Defined in` for these primitives that come from `Error`

Reply:
Got rid of these. Thanks.